jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -677,9 +677,9 @@ public void failed(Throwable x)
         {
             Callback callback = _callback.getAndSet(null);
             if (callback == null)
-                _buffer.releaseAndRemove();
+                _buffer.release();
             else
-                ExceptionUtil.callAndThen(x, t -> _buffer.releaseAndRemove(), callback::failed);
+                ExceptionUtil.callAndThen(x, t -> _buffer.release(), callback::failed);
```

Comment:
I'm surprised this is the only place where we called `releaseAndRemove()`, I'm surprised `HttpOutput` for instance did not.

Reply:
Oh, it does in 12.0 but not in 12.1!

Reply:
Oh no!  We have missed some merges from 12.0!   That needs to be investigated!